src/services/aws_macros.ts

```diff
@@ -39,8 +41,14 @@ type ArgumentTypes<F> = F extends (...args: infer A) => infer B ? [...A, B] : ne
 // break.
 type PromiseReturnType<F> = ArgumentTypes<ArgumentTypes<F>[2]>[1];
 
+// defaults for slow retries
+const SLOW_STRATEGY_RETRIES = 10; // retry a max of 10 times, it will allow to bypass quotas
+const SLOW_STRATEGY_BASE_DELAY = 1000; // wait 1 second - as only delete api has a 5 second quota
+const SLOW_STRATEGY_MAXIMUM_RETRY_DELAY = 20 * 5000;
+
```

Comment:
Same here if they are specific to API Gateway, something like `API_GATEWAY_SLOW_STRATEGY_RETRIES`

Reply:
i want to make it generic